ceph-dev-build/config/definitions/ceph-dev-build.yml

```diff
@@ -79,6 +79,11 @@
             - ../../../scripts/build_utils.sh
             - ../../build/setup_rpm
             - ../../build/build_rpm
+      - trigger-builds:
+        - project: "nfs-ganesha"
+          current-parameters: True
```

Comment:
This might work. If it doesn't you might need to use ``properties-file`` and have ``ceph-dev-build`` write a file with everything in it that ``nfs-ganesha`` might need.